src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -60,6 +59,29 @@ public static async Task RoundTripAsync()
             }
         }
 
+        [Fact]
+        public static async Task WriteLargeJson()
```

Comment:
nit: Rename to RoundTripLargeJsonViaJsonElementAsync

```suggestion
        public static async Task RoundTripLargeJsonViaJsonElementAsync()
```

Reply:
done